### PR TITLE
Missing transcriptions

### LINKED ISSUE #1093: Add missing transcriptions
The diabdat.mpq file has some wav files without transcription from removed quests. They could be very handy for modders or anyone who would like to try to restore the removed content. Or maybe generate a TTS model?

- [X] ~barmaid~ (#1092)
- [x] blacksmith (#1203)
- [X] ~cow!?~ (#1140)
- [X] ~deadguy~  (#1141)
- [X] ~drunk~ (#1146)
- [x] healer (#1212)
- [X] ~narrator~
- [x] pegboy (#1341)
- [x] priest (#1341)
- [x] storyteller (#1341)
- [x] tavtown (#1341)
- [x] witch (#1341)

Very useful references: (thanks to @galaxyhaxz): https://diablo-evolution.net/index.php?pageid=q1b10

https://diablo2.diablowiki.net/D1_Unused_Dialogues

### PR DESCRIPTION
**Pull request by yuripourre**
Here follows the missing transcriptions for Wirt, Tremain, Odgen, Cain and Adria.

Thanks @Tumayto for this great work.

Close #1093

### CODE REVIEW

**Review by Tumayto — APPROVED** [review]
Happy to help @yuripourre. For reference, I originally transcribed these here https://docs.google.com/spreadsheets/d/16RmQqahc2euCpbVRW0itKA-QEUVTAE9tWySzkvca3mE/edit#gid=1786395601

I went through a process of thoroughly listening to each audio file dozens of times and transcribing them. Then I repeated the process to weed out any further mistakes. Additionally, a friend of mine did the same. We saw no issues in regards to grammar, spelling, etc. Some very minor things will likely be open to interpretation/personal preference, but the general integrity is passable. I went as far as to check existing dialog to make sure it leaned towards how the game prefers it to be written.

File: Source/textdat.cpp [hunk 1]
```diff
@@ -696,6 +696,146 @@ const TextDataStruct alltext[] = {
...
+	    1, 5, TSFX_WITCH33 },
+	{ "Remember, the right device in my hands can become quite powerful. |",
+	    1, 5, TSFX_WITCH34 },
+	{ "This is amazing! The mystic energies in this crystal are powerful, but unfocused. If you could find for me a specific staff to channel this power through... |",
+	    1, 5, TSFX_WITCH35 },
+	{ "The crystal holds a magic that is strong, but you need to find the correct staff with which to focus its energy. |",
+	    1, 5, TSFX_WITCH36 },
+	{ "Hand me that staff and stand back! Vita te chlorum an duriam! Ahhhh, it is complete. |",
+	    1, 5, TSFX_WITCH37 },
+	{ "Faith is absolute belief in the unseen. The priest Tremain is from a holy order long asleep in this land. He keeps a promise and a charge issued ages ago, and sustains a union with realms that even my vision cannot reach. He knows much, but not as much as he believes. |",
```
Comment (FitzRoyX) [inline Source/textdat.cpp:837]:
```suggestion
	{ "Faith is absolute belief in the unseen. The priest Tremain is from a holy order long asleep in this land. He keeps a promise and a charge issued ages ago and sustains a union with realms that even my vision cannot reach. He knows much, but not as much as he believes. |",
```

**Review by FitzRoyX — CHANGES_REQUESTED** [review]
-4 or 5 wrongly heard words.
-eye piece is one word
-American spelling of traveler should be used
-many commas removed due to dependent clause
-several commas removed where not needed due to voice actor being dramatic (if it's a long enough pause, ellipses should be used)
-capitalization of titles like lord and archbishop should only happen when in front of a proper name
-changed "mmm" to "hmm" so translators don't think Wirt is hungry

**Comment by yuripourre** [comment]
@FitzRoyX thanks so much for reviewing. I will have some time this week. I will update based on your suggestions and fix conflicts.

**Comment by AJenbo** [comment]
@yuripourre if while you are at it you could also mark them for translation that would be appreciated :)

**Comment by Tumayto** [comment]
Thanks for reviewing @FitzRoyX. I must admit comma placement is something I had trouble with. This was my first time transcribing so it was definitely a learning experience. Good catch on those misheard words. Some are said with such speed that I can still hear them the wrong way if I try.

I capitalized "archbishop" and that one "lords" because it was done with [existing examples of in-game transcription](https://raw.githubusercontent.com/diasurgical/devilutionX/master/Source/textdat.cpp). Ctrl + f "Archbishop" and look through to see what I mean. Cain says "Lords of the underworld" in STORY7. Lords of Hell is debatable as Cain does refer to them that way [officially in a later game](https://diablo.fandom.com/wiki/The_Seven_Lords_of_Hell). If the goal is to go with correction like this instead of maintaining existing quirks, then the in-game transcriptions will definitely need updated as well for consistency.

**Comment by FitzRoyX** [comment]
I haven't looked too hard at the original scripts yet, but I probably could go through them and fix any glaring errors. IIRC the misuse of hyphens annoys me the most about the originals.

I haven't read any style guides that say anything other than lowercase for titles that stand alone. The only exception being verbal address like "Hey there, Archbishop." I also think there's a difference between saying "these two members of the Seven Lords of Hell" and "these two lords of Hell."

**Pull request merged**